Source/Scene/Cesium3DTileset.js

```diff
@@ -349,6 +329,36 @@ define([
         when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
     }
 
+    function selectTileWithTilesetContent(tiles3D, selectedTiles, tile, fullyVisible, frameState, replace) {
+        // 1) If its children are not loaded, load the subtree it points to and then select its root child
+        // 2) If its children are already loaded, select its (root) child since the geometric error of it is
+        //    same as this tile's
+        var childrenLength = tile.children.length;
+        var contentUrl = tile._header.content.url;
+
+        // If the subtree has already been added and child
+        // content requested, select the child and continue
+        if ((childrenLength > 0) && (tile.numberOfChildrenWithoutContent === 0)) {
+            selectTile(selectedTiles, tile.children[0], fullyVisible, frameState);
+            return;
+        } else if (replace) {
+            // Otherwise, select the parent tile, to avoid showing an empty space
+            // Note that tile.parent must always be defined here --
+            // tile should never be the root of the tileset AND
+            // contain another tileset
+            selectTile(selectedTiles, tile.parent, fullyVisible, frameState);
+        }
+
+        // Request the tile's tileset.
+        if (childrenLength === 0 && !tile.tilesetLoading) {
```

Comment:
It's a check to make sure the tiles.json that this tile points to has not already been loaded. If it has been loaded, this tile's number of children will be one (the root of the subtileset specified in corresponding tiles.json).
Currently, tileset tiles will have children after their tiles.json has been loaded. The new subtileset root is appended to the parent's children, and the parent is currently the tile with tileset content.